source/resourceManager.cpp

```diff
@@ -7,6 +9,10 @@ void resourceManager(bool populated, int departmentCounter, string fileStoragePa
     string course[10];
```

Comment:
Whatever this number is - I assume it's an arbitrary limit - it should be saved as a variable, not hard coded. Use const int limit = 10;